src/android/app/src/main/java/es/situm/plugin/PluginHelper.java

```diff
@@ -69,6 +79,16 @@ public void setCommunicationManager(CommunicationManager communicationManager) {
         cmInstance = communicationManager;
     }
 
+    private RealTimeManager getRealtimeManagerInstance() {
+        if (rmInstance == null) {
```

Comment:
Entendo o de facer unha comprobacion extra dentro do synchronized, pero enton non é máis facil facer só a comprobación do synchronized e quitar a outra?